internal/token.go

```diff
@@ -946,12 +955,14 @@ func (z *Tokenizer) hasTag(s string) bool {
 func (z *Tokenizer) readStartTag() TokenType {
 	z.readTag(true)
 	// Several tags flag the tokenizer's next token as raw.
-	c, raw := z.buf[z.data.Start], false
+	c, raw, noExpression := z.buf[z.data.Start], false, false
 	switch c {
 	case 'i':
 		raw = z.startTagIn("iframe")
 	case 'n':
 		raw = z.startTagIn("noembed", "noframes")
+	case 'm':
+		noExpression = z.startTagIn("math")
```

Comment:
👍 There it is.

Reply:
Thanks for commenting, forgot to do my usual pre-review comments.